### DISCUSSION TITLE
Using GitHub Enterprise and can't see invite to Gruntwork GitHub organization

### DISCUSSION THREAD

**Discussion by zackproser** [gruntwork-io/knowledge-base#443] [Help] [answered]
A customer asked: 
> I've been told by Gruntwork support that my GitHub user has been invited to both the 
-  `gruntwork-io` 
-  `gruntwork-clients` (used for Reference Architecture setup and delivery)
but when I log into GitHub with my user, I have no pending invites, and when I attempt to visit either of the above GitHub orgs in my browser, I get a 404. What's wrong?

---

<ins datetime="2022-06-01T17:21:29Z">
  <p><a href="https://support.gruntwork.io/hc/requests/108697">Tracked in ticket #108697</a></p>
</ins>

**Comment by zackproser** [comment, marked as answer]
If you're using GitHub Enterprise and you're unable to see invites to Gruntwork GitHub organizations, even after Gruntwork support has confirmed you've been successfully invited, you'll need to contact your GitHub Enterprise administrator and ask them to grant you permission to view and accept our invites. 

In cases like these, there are policies / settings that your GitHub Enterprise administrator may have configured, which block your ability to view and accept our invitations. When this happens, Gruntwork is unable to assist you further until your GitHub Enterprise administrator has relaxed the constraints on your account.

**Reply by josh-padnick** [reply]
For clarification, this post refers to GitHub's [Enterprise Managed Users](https://docs.github.com/en/enterprise-cloud@latest/admin/identity-and-access-management/using-enterprise-managed-users-for-iam/about-enterprise-managed-users), and we have since learned that, per the docs there:

> Managed user accounts cannot be invited to organizations or repositories outside of the enterprise, nor can the managed user accounts be invited to other enterprises.

This means that you'll need to either:

1. Use your non-Enterprise Managed User to log in (e.g. your personal account), or
2. [Self-host Gruntwork Library](https://docs.gruntwork.io/library/usage/self-hosting)
